pyproject.toml

```diff
@@ -37,16 +38,18 @@ authors = [
     "Nikita Barinov <n.barinov@tinkoff.ru>",
     "Dmitry Bunin <d.a.bunin@tinkoff.ru>",
     "Alexander Chikov <a.p.chikov@tinkoff.ru>",
-    "Artem Makhin <a.makhin@tinkoff.ru>"
+    "Artem Makhin <a.makhin@tinkoff.ru>",
+    "Ivan Nedosekov <ivan-nedd@mail.ru>",
+    "Rodion Petrov <r.petrov@tinkoff.ru>"
 ]
 
 [tool.poetry.dependencies]
-python = ">=3.8.0, <3.11.0"
+python = ">=3.8.0, <3.12.0"
 scikit-learn = ">=0.24, <2"
 pandas = "^1.1"
 catboost = ">=0.21"
 ruptures = "^1.1.5"
-numba = ">=0.53.1,<0.56.0"
+numba = ">=0.53.1,<0.58.0"
```

Comment:
Why do we need this change?

Reply:
since version 0.57 python 3.11 is supported